srcpkgs/python3-Jinja2/template

```diff
@@ -1,8 +1,7 @@
 # Template file for 'python3-Jinja2'
 pkgname=python3-Jinja2
-reverts="3.1.1_1"
-version=3.0.3
-revision=2
+version=3.1.1
+revision=1
```

Comment:
ACK. I thought it was ok to reuse the same revision, since the template will be *identical* to the former `3.1.1_1` one (also, I think the lint step in CI will complain, and I like green checkmarks :rofl:)

Reply:
> (also, I think the lint step in CI will complain, and I like green checkmarks rofl)
lint in CI didn't complain, it seems to be smarter than I thought... :thinking: